inc/saitam.h

```diff
@@ -2370,6 +2460,8 @@ typedef struct _sai_tam_api_t
     sai_remove_tam_counter_subscription_fn         remove_tam_counter_subscription;
     sai_set_tam_counter_subscription_attribute_fn  set_tam_counter_subscription_attribute;
     sai_get_tam_counter_subscription_attribute_fn  get_tam_counter_subscription_attribute;
+    sai_bulk_object_create_fn                      create_tam_counter_subscriptions;
```

Comment:
please use a _tam_ string as part of the type for consistency.

Reply:
I guess it's a convention of SAI to use sai_bulk_object_xxxx.

https://github.com/opencomputeproject/SAI/blob/ad20062eb1f2a24aa79983422e83b921b49c011f/inc/sailag.h#L395-L396
https://github.com/opencomputeproject/SAI/blob/ad20062eb1f2a24aa79983422e83b921b49c011f/inc/sainexthop.h#L335-L338